core/src/main/java/org/keycloak/util/TokenUtil.java

```diff
@@ -38,6 +38,8 @@ public class TokenUtil {
 
     public static final String TOKEN_TYPE_BEARER = "Bearer";
 
+    public static final String TOKEN_TYPE_KEYCLOAK_ID = "Serialized-ID";
```

Comment:
Why do we need a `Serialized-ID` token type? KEYCLOAK-9629 has been deleted from [the issues tracker](https://issues.redhat.com/browse/KEYCLOAK-9629), there is no additional context